report_thesis/src/sections/background.tex

```diff
@@ -316,7 +316,59 @@ \subsubsection{Extra Trees Regressor (ETR)}
 As a tradeoff, \gls{etr} is less interpretable than a single decision tree, as the added randomness can introduce more bias than \gls{rf}.
 However, it often achieves better generalization performance, especially in high-dimensional or noisy datasets.
 
-\subsubsection{Gradient Boosting Regressor (GBR)}
+\subsubsection{Gradient Boosting Regression (GBR)}\label{sec:gradientboost}
+In this section we introduce \gls{gbr} primarily based on \citet{James2023AnIS}.
+\gls{gbr} is an ensemble learning technique that uses a sequence of weak learners, typically decision trees, to build a strong predictive model.
+
+However, in order to explain \gls{gbr}, it is helpful to build on the concepts of ensemble learning and decision trees.
```

Comment:
We need to be really careful with this. We've (or, we will) explained multiple ensemble techniques. It's important to regard the report as a whole, so having a thorough (and good!) description like this about ensembles generally would likely be best served in the introduction to some subsection for the ensemble, tree-based models we use.

And likewise for the theory surrounding decision trees. Most of the other sections we've already merged could actually benefit (IMO) from relying more on the theory they use. That was also one of my comments on the earlier sections, I think on ETR, that we skim over the split decisions.

E.g.:
- Background
    - ... other sections
    - Tree-based ensemble techniques
        - What are ensembles?
        - Decision trees
        - GBR
        - XGB
        - NGB
        - ETR

Or something. And note the order: XGBoost is a highly optimized gradient boosting implementation, with some additional nice features. So it would likely depend on the GBR section. Likewise with NGB.

Thoughts?

Reply:
I agree. I forget which section it is (possibly methodology?) we actually categorize our models like this so it could make sense to make the same categorization in our background. However, I think this should be a separate task after we have written all the sections about the models.

Reply:
Sure, but then I don't think you should spend much more time on this task. Focus on the concepts directly relevant to GBR, like boosting & gradient boosting, and not any of the comments regarding decision trees / ensembles / etc.

Reply:
I made a task in notion for this clean up process.